app/views/guests/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1>Guest Information</h1>
+<h3><%= "#{@guest.name}" %></h3>
+<% @rooms.each do |room| %>
+  <p><%= "Suite: #{room.suite}" %></p>
+  <p><%= "Rate: #{room.rate}" %></p>
+  <p><%= "Hotel: #{room.hotel.name}" %></p>
```

Comment:
Thinking back to the Law of Demeter or the "single-dot" principle, we really don't want to chain on method calls wherever we can avoid it. Create a method on the room model called `hotel_name` and call that instead.